app/views/approaches/show.html.erb

```diff
@@ -23,7 +23,7 @@
                     <div class='user-caption' style='<%= 'margin-top: 0.5em' if @approach.user.avatar.present? %>'>
                       <p class='user-name'><%= link_to "#{@approach.user.display_name}", user_path(@approach.user) %></p>
                       <p class='user-title text-default'><%= @approach.user.display_organization %></p>
-                      <p class='user-link'><%= "#{time_ago_in_words(@approach.created_at)} ago".titleize %></p>
+                      <p class='user-link'><%= "#{t('approaches.modals.show.time', :time => time_ago_in_words(@approach.created_at))}".titleize %></p>
```

Comment:
We've got the `time_ago_in_words` pattern in a lot of places.  In a bid to reduce the number of properties we actually use, is there a way to unify these so they're all pulling from one common property?  It's unlikely/doubtful that the value would differ between views enough to mandate that we create each one individually.